src/main.js

```diff
@@ -1,8 +1,12 @@
-import './style.css';
-import './components/timer/quiz-timer.css';
-import './views/quiz-settings/quiz-settings.css';
+import { renderLeaderboard } from './leaderboard/leaderboard';
 
-import { Router } from './components/router/Router.js';
+renderLeaderboard();
 
-window.addEventListener('load', Router);
-window.addEventListener('hashchange', Router);
+// import './style.css';
+// import './components/timer/quiz-timer.css';
+// import './views/quiz-settings/quiz-settings.css';
+
+// import { Router } from './components/router/Router.js';
+
+// window.addEventListener('load', Router);
+// window.addEventListener('hashchange', Router);
```

Comment:
Router component is available. Do not remove the core lines of code. Implement navigation to the Leaderboard view  using Router